jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java

```diff
@@ -90,6 +109,9 @@
   @Parameter(defaultValue = "true", required = true)
   private boolean enableReproducibleBuilds;
 
+  @Parameter(defaultValue = "Docker", required = true)
+  private ImageFormat imageFormat;
```

Comment:
this doesn't allow variations in capitalizations, right? It must be `Docker` and not `docker`?

Reply:
Yea, it is case sensitive right now. Should we have it be insensitive?